gui/wxpython/mapdisp/statusbar.py

```diff
@@ -380,7 +380,7 @@ def GetPosition(self):
         return self.position
 
     def GetWidget(self):
-        """Returns underlaying winget.
+        """Returns underlaying widget.
```

Comment:
If you are fixing the typo, fix also underlying

Reply:
:-) right. I did not notice.